faq/troubleshooting/rpc-server-unavailable.md

```diff
@@ -45,6 +45,12 @@ Verify the firewall state between RPC Client and RPC Server.
 * If the firewall is disabled, there is no need for any additional changes.
 * If the firewall is enabled, verify that the required ports are also enabled, as stated [here](#internal/faq/miscellaneous/#network-ports-required-for-gathering-the-data) in the section entitled “Network ports required for gathering the data”.
 
+Also, __some users__ had positive experience with the following:
+* Add an inbound firewall rule for "RPC Dynamic Ports" for program "C:\Windows\System32\inetsrv\inetinfo.exe" on the problematic servers.
+* Also add an inbound rule for "RPC Dynamic Ports" for program "%systemroot%\system32\dllhost.exe". 
+
+Limit these rules to the account that runs SPDocKit if you wish. 
+
```

Comment:
I'd make a couple of slight changes to use more formal words: I'd avoid saying _had positive experiences_, instead use something along the lines of _had success with the following_.

I'd also avoid _if you wish_ (it seems very informal to me) and instead maybe something like _You can limit these rules to apply only to the account running SPDocKit._

Reply:
noted. rewritten